docs/guides/error-handling.mdx

```diff
@@ -38,9 +44,7 @@ max-interval = "10s" # max time between retries; default: unset (=interval keeps
 max-attempts = "10" # max number of attempts before terminal error; default: unset (=infinite)
 ```
 
-You can tune this policy to your needs.
-
-Note that all durations should follow the [humantime format](https://docs.rs/humantime/latest/humantime/fn.parse_duration.html).
+You can tune this policy to your needs. Note that all durations should follow the [humantime format](https://docs.rs/humantime/latest/humantime/fn.parse_duration.html).
 
 You can also use a **fixed-delay retry policy**:
```

Comment:
Now that I look at this, I wonder if I should simplify this whole retry policy thing into just exponential retry policy anyway, removing the `type` thing....